### DISCUSSION TITLE
Create multiple gift cards through backend-api in single call

### DISCUSSION THREAD

**Discussion by coderreco** [swellstores/community#550] [Help] [answered]
I am at the tail end of migrating a client from Woocommerce to Swell. One of the final tasks is to migrate all active gift cards with a balance, to Swell. 

Using the backend-api I know that I can create one gift card at a time using `swell.post('/giftcards', {...})`, but is there a way to pass the prepped data as an array of gift cards to create instead of making an api call for each individual card?

Example I have not tried:
```
swell.post('/giftcards', {
[
{// ...card 1 arguments},
{// ...card 2 arguments},
...
]
})
```

Thanks!

**Comment by sags95** [comment, marked as answer]
We do have a `/:batch` endpoint that can be used like this, some details here: https://developers.swell.is/guides/migrate-to-swell#swell-s-migration-endpoint. Avoid using the `$migrate` or `$restore` flags in this case.
